.github/ISSUE_TEMPLATE/test_problem.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Test problem
+about: Report a problem with tests performed by CI (for internal use)
+title: ''
+labels: 'tests'
+assignees: ''
+
+---
+
+**Describe the problem**
+
+
+**Mandatory informations**
+
+ * Branch:
+ * Scenario:
+ * Pipeline ID: 
+ 
```

Comment:
```suggestion
 * Running Date:

```
We may not be able to do it the same day.

Reply:
I think it's not necessary because you can have it with pipeline ID.